lib/event_tracer/dynamo_db_logger.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'time'
+
+module EventTracer
+  class DynamoDBLogger
+    def initialize(buffer = Buffer.new(buffer_size: 0))
```

Comment:
Could we modify this to
```ruby
def initialize(buffer: Buffer.new(buffer_size: 0), app_name: nil)
```

Regarding dynamo db table name, could we use a class variable to set the value (or maybe include it in the initializer above) to avoid constant overwritten warning?

Reply:
Eh? There's no constant defined so it shouldn't complain about constant overwritten I think?

For both app name and dynamo db table name, yes we can actually include it in the initializer. This works fine for the DynamoDBLogger part, but my concern was that we repeatedly have to send these two strings into DynamoDBLogWorker.

Reply:
So if the constant is not defined, the logger wouldn't work? That seems unexpected behavior to me.

If we want to have flexible configuration, could we add real configuration interface instead of asking the app to set constant?

Reply:
Alright! I assume you mean dry-configurable? Made a commit to demonstrate that - let me know if this is correct?